### PR TITLE
[VBLOCKS-2088] feat: update assets

### PR DESCRIPTION
**Pull request by kpchoy**
## Submission Checklist

 - [ ] The `CHANGELOG.md` reflects any **feature**, **bug fixes**, or **known issues** made in the source code
 - [ ] New unit tests and integration tests have beed added
 - [x] Code coverage is more or equal to the previous coverage
 - [x] A visual inspection of the `Files changed` tab was made prior to submitting the pull request ensuring the style guide was followed
 - [ ] [CI pipeline](https://app.circleci.com/pipelines/github/twilio/twilio-voice-react-native-app) is green
 - [x] Included screenshot if necessary

## Description

[VBLOCKS-2088](https://issues.corp.twilio.com/browse/VBLOCKS-2088)
### Breakdown

- update signin page logo
- update twilio logo

## Validation

- visual

## Additional Notes

Screenshot 2023-08-29 at 4 51 34 PM
Screenshot 2023-08-29 at 4 48 21 PM
Screenshot 2023-08-29 at 4 46 12 PM

## Contributing to Twilio

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: app/src/screens/CallInvite/index.tsx [hunk 1]
```diff
@@ -50,7 +50,8 @@ const styles = StyleSheet.create({
     justifyContent: 'space-around',
   },
   logo: {
-    maxHeight: '25%',
+    height: 50,
+    width: 150,
```
Comment (mhuynh5757) [inline app/src/screens/CallInvite/index.tsx:54]: Is this the proper aspect ratio for the image? We typically only glue either the height or the width and let the system handle the scaling so the aspect ratio is preserved.

File: app/src/screens/StackNavigator.tsx [hunk 2]
```diff
@@ -36,19 +32,15 @@ const StackNavigator = () => {
...
   }
 
   return (
-    <Stack.Navigator>
-      <Stack.Screen
-        name="App"
-        options={{ headerShown: false }}
-        component={TabNavigator}
-      />
+    <Stack.Navigator screenOptions={{ headerShown: false }}>
```
Comment (mhuynh5757) [inline app/src/screens/StackNavigator.tsx:42]: Does this change mean we can't navigate back to the app from a call or call invite screen, without using the device's controls?

Comment (kpchoy) [reply app/src/screens/StackNavigator.tsx:42]: Screenshot 2023-08-31 at 3 25 17 PM

I don't think so. I think this just prevents the header from being shown. I dont believe normal apps show headers on every screen.

File: app/src/screens/CallInvite/index.tsx (hunk 1 above)
Comment (mhuynh5757) [reply app/src/screens/CallInvite/index.tsx:54]: Discussed in a call with @kpchoy we're going to leave this as-is for now and file a ticket to use relative measurements everywhere.

**Review by mhuynh5757 — APPROVED** [review]

**Pull request merged**